ui/src/components/chip/__tests__/QChip.spec.js

```diff
@@ -1,154 +1,467 @@
+          mountQChip({
+            props: {
+              outline: true
+            }
+          })
+
+          cy.get('.q-chip.q-chip--outline').should('exist')
         })
       })
 
       describe('(prop): ripple', () => {
-        it.skip(' ', () => {
-          //
+        it('should have a ripple effect when ripple is true', () => {
+          mountQChip({
+            props: {
+              ripple: true
+            }
+          })
+
+          cy.get('.q-chip').click().get('.q-ripple').should('exist')
+        })
+
+        it('should NOT have a ripple effect when ripple is false', () => {
+          mountQChip({
+            props: {
+              ripple: false
+            }
+          })
+
+          cy.get('.q-chip').click().get('.q-ripple').should('not.exist')
         })
       })
     })
   })
 
   describe('Slots', () => {
     describe('(slot): default', () => {
-      it.skip(' ', () => {
-        //
+      it('should display the default slot content', () => {
+        mountQChip({
+          props: {
+            label: undefined
+          },
+
+          slots: {
+            default: 'Default Slot Content'
+          }
+        })
+
+        cy.get('.q-chip').should('have.text', 'Default Slot Content')
       })
     })
   })
 
   describe('Events', () => {
     describe('(event): click', () => {
-      it.skip(' ', () => {
-        //
+      it('should emit click event when clicked and "clickable" is set', () => {
+        const fn = cy.stub()
+
+        mountQChip({
+          props: {
+            clickable: true,
+            onClick: fn
+          }
+        })
+
+        cy.get('.q-chip')
+          .click()
+          .then(() => expect(fn).to.be.calledOnce)
+      })
+
+      it('should NOT emit click event when "clickable" is NOT set', () => {
+        const fn = cy.stub()
+
+        mountQChip({
+          props: {
+            onClick: fn
+          }
+        })
+
+        cy.get('.q-chip')
+          .click()
+          .then(() => expect(fn).to.not.be.called)
       })
     })
 
     describe('(event): update:selected', () => {
-      it.skip(' ', () => {
-        //
+      it('should update selected value when called', () => {
+        const fn = cy.stub()
+
+        mountQChip({
+          props: {
+            selected: false,
+            'onUpdate:selected': fn
+          }
+        })
+
+        cy.get('.q-chip')
+          .click()
+          .then(() => expect(fn).to.be.calledOnce)
+      })
+
+      it('should NOT emit update:selected event selected is NOT applied', () => {
+        const fn = cy.stub()
+
+        mountQChip({
+          props: {
+            'onUpdate:selected': fn
+          }
+        })
+
+        cy.get('.q-chip')
+          .click()
+          .then(() => expect(fn).to.not.be.called)
       })
     })
 
     describe('(event): remove', () => {
-      it.skip(' ', () => {
-        //
+      it('should emit remove event when clicked and removable is applied', () => {
```

Comment:
Sometimes double quotes are used around props names, sometimes not, I'd make it consistent, use them and add "prop" where needed
I'd also document this into the README